src/main/java/org/jivesoftware/openfire/plugin/passwordreset/servlet/admin/PasswordResetSettingsServlet.java

```diff
@@ -154,7 +153,7 @@ private void redirectWithMessage(
     public static final class Dto {
 
         private static final int MAX_PROP_LENGTH = 4000;
-        private static final Set<ChronoUnit> SUPPORTED_UNITS = ImmutableSet.of(
+        private static final List<ChronoUnit> SUPPORTED_UNITS = asList(

```

Comment:
TBH, I'd rather this kept as a Set, and initialised using Set.of(...) - this would require update Java to 11 in build.gradle, but that's the min for Openfire now anyway.